doc/code_style/mom.cpp

```diff
@@ -59,34 +59,32 @@
  * Will be parsed by Doxygen.
  */
 struct another_stupid_type {
-	rofl_t blergh_gnampf;  //!< variables and functions in lowercase
-	                       //   with _ as word separator.
-	epictype42_t løl;      //!< no question, we use UTF-8 encoding.
-	ub0rtype røfëlkøptør;  //!< proper documentation is Doxygen parsable.
+	rofl_t blergh_gnampf;     //!< variables and functions in lowercase
+	                          //   with _ as word separator.
+	epictype42_t løl;         //!< no question, we use UTF-8 encoding.
+	ub0rtype     røfëlkøptør; //!< proper documentation is Doxygen parsable.
 };
 
 
-
 /**
  * Yet another Valve game. Go on, nothing to see here.
  */
 class HalfLife3 : public ValveGame {
-// vvv The access modifier is at the same indent level as the class definition:
+	// vvv The access modifier is at the same indent level as the class definition:
 public:
-	void release_game(const ValveTeam &team,
+	void release_game(const ValveTeam& team,
 	                  uint128_t        passed_time,
-	                  const Engine    *game_engine,
-	                  Internet        *community,
-	                  bool             keep_secret=true) {
+	                  const Engine*    game_engine,
+	                  Internet*        community,
+	                  bool             keep_secret = true) {
 
 		// ^ insert a empty line here to make it more readable.
 
 		// http://www.abstrusegoose.com/440
-		auto gaben = team.get_human("gaben");
+		auto gaben       = team.get_human("gaben");
 		auto min_quality = gaben.get_quality_requirement(Engine::type);
```

Comment:
these alignments should not be done since it requires realigning stuff when variable names change for all other aligned lines, see above.